vulture/core.py

```diff
@@ -491,40 +491,36 @@ def generic_visit(self, node):
 
 
 def _parse_args():
-    def csv(option, _, value, parser):
-        setattr(parser.values, option.dest, value.split(','))
-    usage = """\
-usage: %prog [options] PATH [PATH ...]
-
-Paths may be Python files or directories. For each directory vulture
-analyzes all contained *.py files.
-"""
+    def csv(exclude):
+        return [str(excluded) for excluded in exclude.split(',')]
+    usage = "%(prog)s [options] PATH [PATH ...]"
     version = "vulture {0}".format(__version__)
-    parser = optparse.OptionParser(usage=usage, version=version)
-    parser.add_option(
-        '--exclude', action='callback', callback=csv, metavar='PATTERN',
-        type='string', default=[],
-        help=(
+    parser = argparse.ArgumentParser(prog='vulture', usage=usage)
+    parser.add_argument(
+        'paths', action='store', nargs='+', metavar='PATH', help=(
+            'Paths may be Python files or directories. For each directory'
+            ' vulture analyzes all contained *.py files.'))
+    parser.add_argument(
+        '--exclude', metavar='PATTERN', type=csv, help=(
             'Comma-separated list of paths to ignore (e.g.,'
             ' *settings.py,docs/*.py). PATTERNs can contain globbing'
             ' characters (*, ?, [, ]). Treat PATTERNs without globbing'
             ' characters as *PATTERN*.'))
-    parser.add_option(
-        "--sort-by-size", action="store_true",
-        help="Sort unused functions and classes by their lines of code")
-    parser.add_option('-v', '--verbose', action='store_true')
-    parser.add_option(
-        '--min-confidence', action='store', type='int', default=0, help=(
+    parser.add_argument(
+        '--min-confidence', action='store', type=int, default=0, help=(
             'Minimum confidence (between 0 and 100) for code to be'
             ' reported as unused.'))
-    options, args = parser.parse_args()
-    return options, args
+    parser.add_argument("--sort-by-size", action="store_true", help=(
```

Comment:
Are the brackets for the four `help=()` calls really needed?

Reply:
Given the other option is to use a `\`, I'd say that we should continue using brackets.

Source: https://stackoverflow.com/a/5437652/6426752

Reply:
The brackets are not needed since the strings occur inside brackets anyway.

Reply:
Yes, you are right! (as always) :-)